src/background/mapeo.js

```diff
@@ -20,13 +20,12 @@ class MapeoRPC {
     this.storages = []
     this.config = new Settings(userDataPath)
     this.encryptionKey = this.config.getEncryptionKey()
-    logger.log('got encryptionKey', this.encryptionKey)
+    logger.debug('got encryptionKey', this.encryptionKey)
```

Comment:
Can we not log this? Or just log the first 4 hex chars? This is going to be delicate enough that we want to minimize where we expose it.

Reply:
debug doesn't get written to disk unless you have `debugging` mode turned on -- can do the first 4